diff --git a/dist/vite-plugin.js b/dist/vite-plugin.js
index c1c2b06e3c36bd1a35dee0e77bfad0c8ca4fed0d..e3b2e54d1fdf59d47ae10df64f6f513444eace73 100644
--- a/dist/vite-plugin.js
+++ b/dist/vite-plugin.js
@@ -30,14 +30,14 @@ function configPlugin(options) {
             if (name === VITE_ENVIRONMENT_NAMES.client) {
                 return {
                     build: {
-                        outDir: ".vercel/output/static",
+                        outDir: "dist",
                     },
                 };
             }
             if (name === VITE_ENVIRONMENT_NAMES.server) {
                 return {
                     build: {
-                        outDir: ".vercel/output/render.func",
+                        outDir: ".vercel/output/functions/__server.func",
                         rolldownOptions: {
                             input: {
                                 [MAIN_INPUT]: ENTRYPOINT_VIRTUAL_MODULE,
@@ -102,7 +102,7 @@ export function vercel({ handlerEntrypoint, requestLoggingLevel, ...userOptions
                     // Force ESM?
                     writeFile(join(serverOutDir, "package.json"), JSON.stringify({ type: "module" }), "utf-8"),
                     writeFile(join(serverOutDir, ".vc-config.json"), JSON.stringify({
-                        runtime: "nodejs",
+                        runtime: "nodejs22.x",
                         handler: `${MAIN_INPUT}.mjs`,
                         launcherType: "Nodejs",
                         supportsResponseStreaming: true,
