diff --git a/dist/vite-plugin.js b/dist/vite-plugin.js
index c1c2b06e3c36bd1a35dee0e77bfad0c8ca4fed0d..dfab9fc7363a7d7326a86387c6547388817423ac 100644
--- a/dist/vite-plugin.js
+++ b/dist/vite-plugin.js
@@ -30,14 +30,14 @@ function configPlugin(options) {
             if (name === VITE_ENVIRONMENT_NAMES.client) {
                 return {
                     build: {
-                        outDir: ".vercel/output/static",
+                        outDir: "dist",
                     },
                 };
             }
             if (name === VITE_ENVIRONMENT_NAMES.server) {
                 return {
                     build: {
-                        outDir: ".vercel/output/render.func",
+                        outDir: ".vercel/output/functions/render.func",
                         rolldownOptions: {
                             input: {
                                 [MAIN_INPUT]: ENTRYPOINT_VIRTUAL_MODULE,
